testing/demo/pkg/transfer/celestia_prover_utils.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	proverclient "github.com/celestiaorg/celestia-zkevm-ibc-demo/provers/client"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func getProverSTFKey() (key [32]byte, err error) {
+	info, err := getCelestiaProverInfo()
+	if err != nil {
+		return [32]byte{}, fmt.Errorf("failed to get celestia-prover info %w", err)
+	}
+
+	decoded, err := hex.DecodeString(strings.TrimPrefix(info.StateTransitionVerifierKey, "0x"))
+	if err != nil {
+		return [32]byte{}, fmt.Errorf("failed to decode state transition verifier key %w", err)
+	}
+
+	copy(key[:], decoded)
+	return key, nil
+}
+
+func getProverMembershipKey() (key [32]byte, err error) {
+	info, err := getCelestiaProverInfo()
+	if err != nil {
+		return [32]byte{}, fmt.Errorf("failed to get celestia-prover info %w", err)
+	}
+
+	decoded, err := hex.DecodeString(strings.TrimPrefix(info.StateMembershipVerifierKey, "0x"))
+	if err != nil {
+		return [32]byte{}, fmt.Errorf("failed to decode membership verifier key %w", err)
+	}
+
+	copy(key[:], decoded)
+	return key, nil
+}
+
+func getCelestiaProverInfo() (*proverclient.InfoResponse, error) {
```

Comment:
I guess as a flup we should provider the address as an argument because this function can be used for both the celestia and evm prover

Reply:
oh true, would be a good utility.